src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -311,10 +311,10 @@ private void BuildPreprocessorStateChangeMap()
         /// <summary>
         /// Creates a new syntax tree from a syntax node.
         /// </summary>
-        public static SyntaxTree Create(CSharpSyntaxNode root, CSharpParseOptions? options = null, string path = "", Encoding? encoding = null)
+        public static SyntaxTree Create(CSharpSyntaxNode root, CSharpParseOptions? options = null, string? path = "", Encoding? encoding = null)
         {
 #pragma warning disable CS0618 // We are calling into the obsolete member as that's the one that still does the real work
-            return Create(root, options, path, encoding, diagnosticOptions: null);
+            return Create(root, options, path!, encoding, diagnosticOptions: null);
```

Comment:
If the `Create` calls are followed, it will end up with a `?? string.Empty`. I haven't updated the `Create` overloads per the "BACK COMPAT OVERLOAD -- DO NOT MODIFY". But I think it might be safe to update them, there will be no breaking change.

Reply:
We should update it, that's not an API breaking change.